Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -63,8 +63,8 @@ public static void UnhookEvents()
                     info.Subclass.FocusChange -= FocusDispatcher;
                     info.Subclass.Dispose();
                 }
-                SafeComWrappers.VBA.VBProjects.DetatchEvents();
-                SafeComWrappers.VBA.VBComponents.DetatchEvents();
+                //SafeComWrappers.VBA.VBProjects.DetatchEvents();
+                //SafeComWrappers.VBA.VBComponents.DetatchEvents();
             }
```

Comment:
This destroys the proper shutdown order: we want to detach all events before we start tearing down RD. (There is no guarantee in which order thisgs will get disposed on disposal of the IoC container. So, we do not want anything from the outside kick off anything while  we tear down.)